### ISSUE TITLE
Mixup for multi-label classification

### ISSUE THREAD

**Issue by bearpelican** [fastai/fastai_dev#269]
Trying to use mixup for the rsna challenge.

Errors on toy examples here:
https://github.com/bearpelican/fastai_dev/blob/mixup_multilabel/dev/19_callback_mixup.ipynb

[Unsqueezing](https://github.com/fastai/fastai_dev/blob/master/dev/local/callback/mixup.py#L37) seems to work for the unflattened version

**Comment by sgugger** [comment]
Oh, I'm guessing the loss function returns the full tensors and there is a broadcasting error. Probably requires the same line as for the xs

**Comment by sgugger** [comment]
I've changed the callback to only use stacking ys with CrossEntropyFlat or LabelSmoothingCrossEntropy, in all other cases, targets are linearly interpolated like the xs, which should fix your problem. Let me know if there is still some bugs left.
